examples/src/main/java/alluxio/cli/UnderFileSystemContractTest.java

```diff
@@ -125,16 +123,20 @@ private void loadAndRunTests(Object operations, String testDir) throws Exception
       for (Method test : tests) {
         String testName = test.getName();
         if (testName.endsWith("Test")) {
-          LOG.info("Running test: " + testName);
+          System.out.printf("Running test: %s...", testName);
+          Thread thread = CommonUtils.createProgressThread(System.out);
+          thread.start();
           try {
             test.invoke(operations);
           } catch (InvocationTargetException e) {
             if (mUfs.getUnderFSType().equals(S3_IDENTIFIER)) {
               logRelatedS3Operations(test);
             }
             throw new IOException(e.getTargetException());
+          } finally {
+            thread.interrupt();
```

Comment:
do we always want to interrupt the thread?

Reply:
Actually we do need to thread to print dot. 
The thread will be created and interrupted in a short period if we run tests against Hdfs or local UFS.

Reply:
Removed the print dot thread.